src/basho_bench_stats.erl

```diff
@@ -67,6 +67,8 @@ op_complete(Op, {ok, Units}, ElapsedUs) ->
             gen_server:cast({global, ?MODULE}, {Op, {ok, Units}, ElapsedUs});
         false ->
             folsom_metrics:notify_existing_metric({latencies, Op}, ElapsedUs, histogram),
+            folsom_metrics:notify_existing_metric({overall_latencies, Op}, ElapsedUs, histogram),
+            folsom_metrics:notify_existing_metric({overall_units, Op}, {inc, Units}, counter),
```

Comment:
What does `{overall_units, Op}` provide us? I'm not sure we need this, seems like we could get this same data by summing the `n` column values from the other data sets. Or is there something else this provides?